src/main/java/com/bootcamp/demo/service/assembler/ScooterAssembler.java

```diff
@@ -0,0 +1,11 @@
+package com.bootcamp.demo.service.assembler;
+
+import com.bootcamp.demo.model.Scooter;
+import com.google.cloud.firestore.QueryDocumentSnapshot;
+
+public class ScooterAssembler {
```

Comment:
I'm not sure if we need this class since the mapping is straightforward and could be called inline.